flang/test/Lower/intrinsic-procedures/command_argument_count.f90

```diff
@@ -0,0 +1,10 @@
+! RUN: flang-new -fc1 -emit-fir %s -o - | FileCheck %s
+! RUN: flang-new -fc1 -fdefault-integer-8 -emit-fir %s -o - | FileCheck --check-prefixes=CHECK,CHECK-64  %s
+
+! CHECK-LABEL: argument_count_test
+subroutine argument_count_test()
+integer :: arg_count_test
+arg_count_test = command_argument_count()
+! CHECK: %[[i1:.*]] = fir.call @_FortranAArgumentCount() : () -> i32
```

Comment:
`i1` is a data type in LLVM IR and MLIR. To avoid confusion, I would rename it. Unless I'm missing something here?

Reply:
Originally I was thinking it in terms of integer-1, integer-2, etc. Changed to to an actual variable name 'argumentCount'.